src/pages/bootstrap/bootstrap-carousel/index.md

```diff
@@ -121,7 +124,25 @@ g) `carousel-caption` :- This class let you add captions to each slide.
 
 Note: Add `class="carousel-caption"` for each item.
 
+### Usage
+#### 1) Via data attributes
+The `data-ride="carousel"` attribute is used to mark a carousel as animating starting at page load.
+`data-slide` accepts the keywords `prev` or `next`, which alters the slide position relative to its current position. 
+#### 2) Via JavaScript
+Call carousel manually with:
+
+`$('.carousel').carousel()`
+
+### Options
+Options can be passed via data attributes or JavaScript. For data attributes, append the option name to `data-`, as in `data-interval=""`.
+
+Some frequently used options are:
+
+* interval
+* pause
+* ride
+* wrap
 
-#### Additional Details : <a href='https://getbootstrap.com/docs/4.0/components/carousel/' target='_blank' rel='nofollow'>Bootstrap Carousel</a>
+## Additional Details : <a href='https://getbootstrap.com/docs/4.0/components/carousel/' target='_blank' rel='nofollow'>Bootstrap Carousel</a>
```

Comment:
Remember to use markdown links.

Reply:
Moreover, we use bullet point lists for informative links at the end of the article.